rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -139,7 +139,7 @@ def __init__(
         rules = self._recursively_initialize_decoders(self.chain_modules_root)
         self.rules += rules
         # Sort post decoding rules by priority (which is the first element of the tuple)
-        self.rules.post_decoding_rules.sort(key=lambda x: x[0], reverse=True)
+        self.rules.post_decoding_rules.sort(key=lambda x: x[0])
```

Comment:
Wait ... isn't this change breaking all the existing priorities for post-decoding rules?. @nebolax put it there for a reason, no?

Reply:
Currently I think it won't break anything since currently all post-decoding rules have priority of `0`. So I would say it is just more a convenience question for the future -- whether decoders with lower or with higher priority should run first.

Reply:
I didn't change it for convenience. It was to match what the docstring was saying https://github.com/rotki/rotki/blob/6a2a706c295e4a307c3b1f91baf431f52f5dc457/rotkehlchen/chain/evm/decoding/interfaces.py#L58

As @nebolax said I checked that this change doesn't affect any other decoder